examples/action_dann_lightn/requirements.txt

```diff
@@ -0,0 +1,13 @@
+# https://github.com/kornia/kornia/blob/master/requirements.txt
+# Versions used in development
+
+# core dependencies
+pytorch==1.7.0
+pytorch-lightning==1.0.2
+numpy==1.19.1
+Python==3.8
+
+# utils
+torchvision==0.8.1
+torchsummary==1.5.1
+yacs==0.1.8
```

Comment:
I think I need one for digits_dann. Have you tested digits_dann and is it proper/safe to put the same requirements as action_dann?

Reply:
`digit_dann` can also run in this environment, and I will update and unify both `requirements.txt` files. But we may need to unify our Python and PyTorch version for the whole PyKale at first. Then I can upgrade or downgrade this requirement.